packages/notification-service/src/firebase.ts

```diff
@@ -130,11 +130,11 @@ export function getPendingRequests() {
   return pendingRequests
 }
 
-export function setPaymentRequestNotified(uid: string) {
-  database.ref(`/pendingRequests/${uid}`).update({ notified: true })
+export async function setPaymentRequestNotified(uid: string) {
+  await database.ref(`/pendingRequests/${uid}`).update({ notified: true })
```

Comment:
Ditto as above. Lets replace await with return here and make this function not async